<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<html><!-- InstanceBegin template="/Templates/obigriffith_template.dwt" codeOutsideHTMLIsLocked="false" -->
<!-- DW6 -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<!-- InstanceBeginEditable name="doctitle" -->
<title>obigriffith.org: research</title>
<!-- InstanceEndEditable --><link rel="stylesheet" href="styles/2col_leftNav.css" type="text/css">
<!-- InstanceBeginEditable name="head" --><!-- InstanceEndEditable -->
</head>
<!-- The structure of this file is exactly the same as 2col_rightNav.html;
     the only difference between the two is the stylesheet they use -->
<body> 
<div id="masthead"> 
  <h1 align="left" id="siteName"><a href="index.html" target="_self">obigriffith.org</a></h1> 
  <div id="globalNav"> 
    <a href="index.html">home</a>| <a href="research.html"> research</a>| <a href="funding.html"> funding</a>| <a href="publications.html">publications</a>| <a href="teaching.html">teaching</a>| <a href="CV.html">CV</a>| <a href="contact.html">contact</a></div> 
</div> 
<!-- InstanceBeginEditable name="main_body" -->
<div id="content">
    <h1>Research</h1><br>
    <h2>Current - The Genome Institute, Washington University  School of Medicine</h2>
    <p>I am  Assistant Professor (<a href="http://internalmed.wustl.edu/">Department of Medicine</a>, <a href="http://oncology.wustl.edu/">Division of Oncology</a> and <a href="http://genetics.wustl.edu/">Department of Genetics</a>) and Assistant Director at <a href="http://genome.wustl.edu/">The McDonnell Genome Institute</a>.  My lab's research interests are best described as cancer   informatics and clinical statistics.  Specifically, I use computational   methods for the analysis of large cancer datasets at the molecular and   genomic level. I am engaged in a number of large-scale tumor sequencing projects on liver and breast cancer, investigating primary, relapse and drug resistant tumors. To this end I am working with others to develop  end-to-end pipelines for clinical cancer sequencing that automate state-of-the-art methods for sequence alignment, somatic variation detection, RNA sequence analysis, and the integration of these data types into user-friendly reports of the most clinically relevant genome and transcriptome changes in a tumor or group of tumors. To aid in this effort I have  developed a database and web tool for interrogating the druggable genome (<a href="http://www.dgidb.org">www.dgidb.org</a>). Projects under development include the Database of Curated Mutations (<a href="http://www.docm.info">www.docm.info</a>) and the CIViC resource for Clinical Interpretation of Variants in Cancer (<a href="http://www.civicdb.org">www.civicdb.org</a>). Visit the <a href="publications.html">publications</a> section for papers describing these projects in greater detail.</p>
    <h2>Past - Lawrence Berkeley National Laboratory</h2>
<p> I was a  post-doctoral fellow (Cancer and DNA Damage Responses, <a href="http://www.lbl.gov/lsd/index.html">Life Sciences Division</a>) at <a href="http://www.lbl.gov">Lawrence Berkeley National Laboratory</a> under the supervision of <a href="http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/basic-science-departments/biomedical-engineering/people/joe-gray.cfm">Dr. Joe Gray</a> and <a href="http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/basic-science-departments/molecular-and-medical-genetics/people/primary-faculty/paul-spellman.cfm">Dr. Paul Spellman</a>. There, I worked on developing methods for a systems biology approach to the analysis of next-generation sequencing data (e.g. Illumina) for metastatic breast cancer and its treatment. I  helped to develop the Alexa-seq platform (<a href="http://www.alexaplatform.org">www.alexaplatform.org</a>)  for analysis and visualization of RNA-seq data and  applied it to a number of cell line and primary  tumor datasets with the goal of identifying omic predictors of drug response in  breast cancer (<a href="http://www.alexa-seq-lbl.org">www.alexa-seq-lbl.org</a>). I also developed a  predictor for breast cancer recurrence that is <a href="http://www.lbl.gov/tt/techs/lbnl3199.html">currently under commercial development</a> as a faster and cheaper alternative to existing products. Visit the <a href="publications.html">publications</a> section for papers describing these projects in greater detail. </p>
    <h2>Past - Genome Sciences Centre, BC Cancer Research Centre</h2>
  <p>I completed a PhD and post-doctoral fellowship (<a href="http://www.medgen.ubc.ca/">Dept. of Medical Genetics</a>, <a href="http://www.ubc.ca/" target="_blank">University of British Columbia</a>) at the <a href="http://www.bccrc.ca/">British Columbia Cancer Research Centre</a> and <a href="http://www.bcgsc.ca" target="_top">Genome Sciences Centre</a> (GSC) under the supervision of <a href="http://www.bccrc.ca/gsc/people_sjones.html" target="_top">Dr. Steven Jones</a> (Associate Director, Head of Bioinformatics). My research focused on understanding how   changes in the regulatory sequences of DNA contribute to cancer   development and progression.   I developed novel algorithms   for investigating cancer expression data in the areas of differential   coexpression and subspace clustering. Complementary to these approaches   was the curation and collection of known regulatory sequences and   mutations.  To this end, I developed ORegAnno (<a target="blank" href="http://www.oreganno.org">www.oreganno.org</a>),   an open access database and curation system for regulatory sequences   and polymorphisms. This system has captured the experimental results of   hundred of studies of genetic and epigenetic regulation.  I also worked on the identification of molecular markers   at the DNA, RNA and protein level that are useful for diagnosis and   prognosis of cancer.  Using bioinformatics methods, I  conducted   extensive meta-analyses of expression profiling studies for thyroid   cancer.  This allowed the identification and ranking of high-confidence   differentially expressed genes between cancer and non-cancer states.   Such candidates were then validated by immunohistochemistry and tissue   microarray analysis on a cohort of over two-hundred human thyroid tumour   patient samples (in collaboration with <a href="http://surgery.med.ubc.ca/people/swiseman/">Dr. Sam Wiseman</a>, St. Paul's   Hospital). This allowed the development of a panel of markers that can   classify malignant versus benign thyroid neoplasms with high accuracy   (&gt;90%) and thus has potential utility as a diagnostic tool for   thyroid cancer.  The approach was also extended to colon and rectal   cancer. Visit the <a href="publications.html">publications</a> section for papers describing these projects in greater detail.</p>
</div>
<!-- InstanceEndEditable -->
<!--end content --> 

<div id="navBar"> 
  <div class="relatedLinks">
    <h3>Research</h3> 
    <ul> 
      <li><a href="http://genome.wustl.edu/" target="_top">MGI</a></li> 
      <li><a href="http://www.lbl.gov" target="_top">LBNL</a></li> 
      <li><a href="http://www.bcgsc.ca" target="_top">GSC</a></li> 
      <li><a href="http://civicdb.org" target="_top">CIViC</a></li>
      <li><a href="http://docm.info" target="_top">DoCM</a></li>
      <li><a href="http://dgidb.org" target="_top">DGIdb</a></li>
      <li><a href="https://www.synapse.org/#!Synapse:syn2179898" target="_top">Rtoolbox</a></li>
      <li><a href="http://www.AlexaPlatform.org" target="_top">ALEXA</a></li>
      <li><a href="http://www.alexa-seq-lbl.org" target="_top">ALEXA-LBL</a></li>
      <li><a href="http://www.oreganno.org" target="_top">ORegAnno</a></li>
      <li><a href="http://www.cisred.org" target="_top">cisRED</a></li>
      <li><a href="http://www.bcgsc.ca/platform/bioinfo/ge/thyroid/" target="_top">thyroid cancer</a></li>
      <li><a href="http://www.bcgsc.ca/platform/bioinfo/ge/kiwi" target="_top">subspace clustering</a></li> 
      <li><a href="http://www.bcgsc.ca/gc/bomge/coexpression/" target="_top">coexpression</a></li>
    </ul> 
  </div> 
  <div class="relatedLinks"> 
    <h3>Colleagues</h3> 
    <ul> 
      <li><a href="http://www.malachigriffith.org/">Malachi Griffith</a></li>
      <li><a href="http://sulab.org/the-team/benjamin-good/">Benjamin Good</a></li> 
      <li><a href="http://cs.txstate.edu/~jg66/">Byron Gao</a></li> 
      <li><a href="http://montgomerylab.stanford.edu/index.html">Stephen Montgomery</a></li>
      <li><a href="http://www.linkedin.com/in/anneleendaemen">Anneleen Daemen</a></li>
      <li><a href="http://www.linkedin.com/pub/laura-heiser/58/27a/67">Laura Heiser</a></li>        
      <li><a href="http://genome.wustl.edu/people/wilson_richard">Rick Wilson</a></li>
      <li><a href="http://genome.wustl.edu/people/mardis_elaine">Elaine Mardis</a></li>      
      <li><a href="http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/basic-science-departments/biomedical-engineering/people/joe-gray.cfm">Joe Gray</a></li> 
      <li><a href="http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/basic-science-departments/molecular-and-medical-genetics/people/primary-faculty/paul-spellman.cfm">Paul Spellman</a></li> 
      <li><a href="http://www.bcgsc.ca/faculty/sjones">Steve Jones</a></li>
      <li><a href="http://www.surgery.ubc.ca/faculty/swiseman.html">Sam Wiseman</a></li>      
    </ul> 
  </div> 
  <div class="relatedLinks"> 
    <h3>Favourites</h3> 
    <ul>
      <li><a href="http://bioinformatics.ca/" target="_top">Canadian Bioinformatics Workshops</a></li>
      <li><a href="http://www.biostars.org/" target="_top">BioStar</a></li>
      <li><a href="http://www.codecademy.com/" target="_top">Codecademy</a></li>
    </ul> 
  </div>
</div> 

<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-3237550-1");
pageTracker._initData();
pageTracker._trackPageview();
</script>
</body>
<!-- InstanceEnd --></html>
